universe/Pathfinder.cpp

```diff
@@ -838,6 +839,13 @@ namespace {
         if (fleet)
             return std::make_pair(fleet->PreviousSystemID(), fleet->NextSystemID());
 
+        if (auto field = std::dynamic_pointer_cast<const Field>(obj))
```

Comment:
the `auto field = ` is probably unnecessary and likely to produce an unused variable warning.

Reply:
fixed.